src/common/product_models/heatmap_common_models.py

```diff
@@ -3,25 +3,29 @@
 """
 
 from enum import Enum
+
 from pydantic import BaseModel, Field
 
 from src.common.product_models.common_models import SubsetProcessStates
 
-
 # these fields need to match the fields in the models below.
 FIELD_HEATMAP_DATA = "data"
 FIELD_HEATMAP_MIN_VALUE = "min_value"
 FIELD_HEATMAP_MAX_VALUE = "max_value"
 FIELD_HEATMAP_COUNT = "count"
-FIELD_HEATMAP_CELL_VALUE = "val"
+FIELD_HEATMAP_KBASE_ID = 'kbase_id'
```

Comment:
This should use the same variable for `kbase_id` as everything else since it needs to match across data products

Reply:
👍

Reply:
:+1: